vetiver-testing/setup-rsconnect/rstudio-connect.gcfg

```diff
@@ -13,6 +13,7 @@ DefaultUserRole = publisher
 
 [Python]
 Enabled = true
+Executable = /opt/python/9.9.9/bin/python
```

Comment:
Intentionally triggering an error to confirm CI shows full Connect logs. To be removed.